docs/components/time_pickers.md

```diff
@@ -0,0 +1,329 @@
+<!--docs:
+title: "Material time pickers"
+layout: detail
+section: components
+excerpt: "Time pickers help users select and set a specific time."
+iconId: 
+path: /catalog/time-pickers/
+-->
+
+# Time pickers
+
+[Time pickers](https://material.io/components/time-pickers) help users select and set a specific time. 
+
+![Time picker with input and dial set to 7PM](assets/time_pickers/TimePicker_hero.png)
+
+
+## Contents
+
+**Note to developers: review the TOC contents to ensure that all rel links apply to your platform dev doc**
+
+* [Using time pickers](#using-time-pickers)
+* [Making time pickers accessible](#making-time-pickers-accessible)
+* [Mobile time pickers](#mobile-time-pickers)
+* [Mobile time input pickers](#mobile-time-input-pickers)
+* [Theming time pickers](#theming-time-pickers)
+
+## Using time pickers
+
+Time pickers allow users to enter a specific time value. They can be used for a wide range of scenarios.  
+Common use cases include:
+* Setting an alarm
+* Scheduling a meeting
+
+Mobile time pickers are displayed in dialogs and can be used to select hours, minutes, and a period of time
+
+
+## Making time pickers accessible 
+
+Flutter's APIs support accessibility setting for large fonts, screen readers, and sufficient contrast. For more information, go to Flutter's [accessibility](https://flutter.dev/docs/development/accessibility-and-localization/accessibility) and [internationalization](https://flutter.dev/docs/development/accessibility-and-localization/internationalization) pages.
+
+
+## Types
+
+_**The following types are for non-iOS platforms**_
+
+There are two types of time pickers: 1\. [mobile time pickers](#mobile-time-pickers), and 2\. [mobile time input pickers](#mobile-time-input-pickers):
+
+![Time picker composite. Image 1 shows a digital input over a 12 hour dial input. Image 2 shows a digital input](assets/time_pickers/TimePicker_types.png)
+
+### Mobile time pickers
+
+Mobile time pickers can be used to specify time down to hours and minutes in a time period. They are displayed in a dialog.  
+
+* Common use cases include: 
+* Setting an alarm
+* Scheduling a meeting
+
+Mobile time pickers allow selection of hours and minutes. They aren’t ideal for selecting granular amounts of time, such as milliseconds for a stopwatch application.
+
+
+#### Mobile time pickers example
+
+
+`showTimePicker`
+* [Class definition](https://api.flutter.dev/flutter/material/showTimePicker.html)
+* [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/time_picker.dart)
+- [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/time_pickers/regular&gh_ref=develop)
+
+The following example shows a time picker.
+
+![Time picker](assets/time_pickers/time_picker_example.png)
```

Comment:
Update alt-text with:

"Time picker with dial set to 7 and input showing 7:15AM"